src/server/run_sched.c

```diff
@@ -302,8 +302,7 @@ contact_sched(int cmd, char *jobid, pbs_net_t pbs_scheduler_addr, unsigned int p
 
 	(void)sigaction(SIGALRM, &oact, NULL);	/* reset handler for SIGALRM */
 #endif
-
-	static int priority_flag = 1;
+	const int priority_flag = 1;
```

Comment:
We usually use a macro for such things instead of a constant

Reply:
done